lix/source-code/client/src/api.ts

```diff
@@ -9,6 +9,8 @@ type Author = {
 	timezoneOffset?: number
 }
 
+export type LixError = Error & { response?: { status?: number } }
```

Comment:
I recommend to extend the Error class to stay compliant with JS and increase interop. Take a look at the errors in `@inlang/sdk`.

```ts
class LixError extends Error {
  // ...
}

```

Reply:
Second this